.travis.yml

```diff
@@ -7,11 +7,15 @@ env:
     - "ARTIFACTS_S3_BUCKET=vtest.musescore.org"
     - secure: "lVj+9BBtJIjW3CwfXstvNyYVn0AkXEwTyiPmp4BprcheP78WIqZNC0uG2RjG9MgyHbZkprE7zRdqR9YPWTitg+XYkkD6+jPHRO+PQFLARuiTAX9yhUO53yQQZC2wMkQ#+bFuZsFmz1rfAsPHx3bXeZAMsz+Qnh8D2yIqqV7qxwWw="
     - secure: "L+66yQZIZJTyIAfrG89ncKIkMAr4+UvaOZMsd420OSdnEH9kpdm5Kws8rG0VVLAtqhfQfi3K9DuC8Ub7IiXqil//h+I9WJ2LHKirWK0m/MkhTFC6hfi0uSnQCX/jud3Keewxf3ovgiKQvIw6VR37UC50YJM9+KhZKtsNYhGAdos="
+    # Branches for uploads of Linux nightly builds & stable release AppImages
+    - "APPIMAGE_UPLOAD_BRANCHES=\"master 2.0.3\""
+    # AppImages are built on all branches, but only uploaded on those listed
```

Comment:
Indeed it would! But the build has to start anyway to check which branch we are on so I thought it might as well continue. There are supposed to be [ways](https://docs.travis-ci.com/user/customizing-the-build/#Building-Specific-Branches) to [prevent the build from starting at all](https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix) but I couldn't get them to work in a way that would satisfy our needs.